test/built-ins/Array/fromAsync/this-constructor-operations.js

```diff
@@ -60,7 +60,6 @@ asyncTest(async function () {
 
   // Note https://github.com/tc39/proposal-array-from-async/issues/35
```

Comment:
We can delete this comment now, too, since the fix makes it obsolete.

Reply:
Done. Thanks for pointing that out.